### DISCUSSION TITLE
Display brightness in smaller increments

### DISCUSSION THREAD

**Discussion by frederiktwiehaus** [pqrs-org/KE-complex_modifications#1162] [Ideas]
Hi can anybody tell me how I could use Karabiner Elements to have
Option+Shit+F1 pressed when I press F1
same for F2
to adjust the brightness automatically in smaller increments :)

Cheers

**Comment by MuhammedZakir** [comment]
Use complex modification. See:
- https://karabiner-elements.pqrs.org/docs/manual/configuration/configure-complex-modifications/
- https://karabiner-elements.pqrs.org/docs/json/
- https://karabiner-elements.pqrs.org/docs/json/typical-complex-modifications-examples/

E.g.
```json
{
  "description": "{F1,F2} -> Shift+Option+{F1,F2}",
  "manipulators": [
    {
      "type": "basic",
      "from": {
        "key_code": "f1",
        "modifiers": {
          "optional": ["caps_lock"]
        }
      },
      "to": [{
        "key_code": "f1",
        "modifiers": ["shift", "option"]
      }]
    },
    {
      "type": "basic",
      "from": {
        "key_code": "f2",
        "modifiers": {
          "optional": ["caps_lock"]
        }
      },
      "to": [{
        "key_code": "f2",
        "modifiers": ["shift", "option"]
      }]
    }
  ]
}
```
